File: src/types/NewContainerRequest.ts
interface NewContainerRequest {
  container_name: string;
  home_directory: string;
  unshare_devsys: boolean;
  unshare_groups: boolean;
  unshare_ipc: boolean;
  unshare_netns: boolean;
  unshare_process: boolean;
  unshare_all: boolean;
}

File: src-tauri/src/structs/new_container_request.rs
#[derive(Clone, Debug,serde::Serialize, serde::Deserialize)]
pub struct NewContainerRequest {
  pub container_name: String,
  pub home_directory: String,
  pub unshare_devsys: bool,
  pub unshare_groups: bool,
  pub unshare_ipc: bool,
  pub unshare_netns: bool,
  pub unshare_process: bool,
  pub unshare_all: bool,
}
